docs/custom-implementation.md

```diff
@@ -0,0 +1,45 @@
+# Custom Implementation
+
+!!! tip
+
+    Install Git, Maven and Docker Compose
+
+HIS systems, like Ozone, often need to be heavily customized based on local needs. Ozone facilitates implementer's providing custom implementation configurations via a series of tools built on Apache Maven. The starting-point for a custom implementation is the Ozone Maven Archetype which creates a skeleton project that serves as a base for any particular customizations an implementation needs.
+
+## Getting Started
+
+To get started, use Maven's archetype tools to generate a new Ozone implementation project. The command for doing this is:
+
+```bash
+ mvn archetype:generate -DarchetypeArtifactId=ozone-distro-archetype -DarchetypeGroupId=com.ozonehis
+```
+
+This will prompt you for several key variables for your implementation:
+
+* **distributionName:** A user-friendly name for your distribution. For example, an Ozone implementation for the country of Gruzinia might use "Gruzinia" as the distribution name.
```

Comment:
I would suggest that archetype doesn't add the `Ozone` word to the `distribution name` as below. 

![image](https://github.com/ozone-his/ozone-docs/assets/36139599/e78385fe-9b1a-4944-a710-40f760666f0d)

Reply:
Not always.

When distributions don't have a proper product name, yes we should put `Ozone` in front. When they do, then they should be able to not have to have `Ozone`.
KenyaHMIS distro for instance is named `KenyaHMIS` not `Ozone KenyaHMIS`.
But for Haiti, well, we don't have a good product name for it, so it's just `Ozone Haiti`

> **distributionName:** A user-friendly name for your distribution. For example, an Ozone implementation for the country of Gruzinia might use "Ozone Gruzinia" as the distribution name.